MAUI/ImageEditor/shape.md

```diff
@@ -0,0 +1,206 @@
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="C#" %}
+
+    private void OnAnnotationSelected(object sender, AnnotationSelectedEventArgs e)
+    {
+        if (e.AnnotationSettings is ImageEditorShapeSettings shapeSettings)
+        {
+            shapeSettings.Color = Colors.Black;
+        }
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Add shape on initial loading
+
+Annotate a shape on image loading using the [`Imageloaded`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ImageLoaded) event.
+
+{% tabs %}
+
+{% highlight xaml tabtitle="XAML" %}
+
+    <imageEditor:SfImageEditor x:Name="imageEditor" Source="image.png" ImageLoaded = "OnImageLoaded" />
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="C#" %}
+
+    private void OnImageLoaded(object sender, EventArgs e)
+    {
+        this.imageEditor.AddShape(AnnotationShape.Circle);
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Add shape with manual bounds
+
+Shapes can be added by user-defined view bounds. The [`Bounds`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorAnnotationSettings_Bounds) are treated as ratio values of image width and height, so you have to specify bounds rectangle values in the range of 0.1
+
+{% tabs %}
+{% highlight C# tabtitle="C#" %}
+
+   imageEditor.AddShape(AnnotationShape.Arrow, new ImageEditorShapeSettings() 
+   { 
+    Bounds=new Rect(0.1,0.1,0.5,0.5)
+   });
+
+{% endhighlight %}
+{% endtabs %}
+
+## Restrict shape drag and resize
+
+To restrict the drag action on a shape, set the [`AllowDrag`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorAnnotationSettings_AllowDrag) property to `false`.
+
+{% tabs %}
+{% highlight C# tabtitle="C#" %}
+
+    imageEditor.AddShape(AnnotationShape.Circle, new ImageEditorShapeSettings() { AllowDrag = false });
+
+{% endhighlight %}
+{% endtabs %}
+
+To restrict the resize action on a shape, set the [`AllowResize`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorAnnotationSettings_AllowResize) property to `false`.
+
+{% tabs %}
+{% highlight C# tabtitle="C#" %}
+
+    imageEditor.AddShape(AnnotationShape.Circle, new ImageEditorShapeSettings() { AllowResize = false });
+
+{% endhighlight %}
+{% endtabs %}
+
+## Freehand Draw
+
+The image editor control allows you to create freehand drawings with customizable settings. The [`AddShape`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_AddShape_Syncfusion_Maui_ImageEditor_AnnotationShape_Syncfusion_Maui_ImageEditor_ImageEditorShapeSettings_) method enables the canvas view, in which you can draw objects.
```

Comment:
The image editor control allows you to create freehand drawings such as signature, pen drawing with c

Reply:
Updated a content as you mentioned